event/event.go

```diff
@@ -296,7 +295,7 @@ func (ev *event) Handler() http.Handler {
 			return lab.MaximumLabsErr
 		}
 
-		ev.teamQueue <- *t
+		ev.teamQueue <- t
```

Comment:
If processQueue exists without Assigning a lab the [registerIntercept](https://github.com/aau-network-security/haaukins/blob/master/svcs/ctfd/interception.go#L381) will save team to the teamstore without the changes from assignLabs.

At that point wouldn't `t` point to a team in [registerIntercept](https://github.com/aau-network-security/haaukins/blob/master/svcs/ctfd/interception.go#L381) and not the one in the team store?

Reply:
But in that case, should not we have consider the situations where we have exit in `processQueue` ?

Reply:
Yes you mean then processQueue can't find any labs?

But i'm not sure what processQueue should do then.